selinux/defaults/main.yml

```diff
@@ -0,0 +1 @@
+selinux_state: disabled
```

Comment:
I don't have experience with SELinux. For the case of the ansible-rails-example it seems that it would sufficient to add a single rule to allow the webserver to access the appserver's port.

I'll check it out...